source/localizable/smart_cart/fbs_api.html.md.erb

```diff
@@ -72,6 +73,24 @@ curl -X GET https://<%= settings.api_domain %>/merchants/ecommerce/fbs/purchase_
 
 <%= render_recording :merchants_fbs_purchase_orders_index %>
 
+### Create a new purchase order
+
+To retrieve a shop's Fulfilled by Skroutz purchase orders.
+
+<pre class="terminal">
+  POST /merchants/ecommerce/fbs/purchase_orders
+</pre>
+
+#### Example
+<pre class="terminal">
+curl -X POST https://<%= settings.api_domain %>/merchants/ecommerce/fbs/purchase_orders \
+  -H 'Accept: application/vnd.<%= settings.site_name %>+json; version=3.0' \
+  -H 'Authorization: Bearer your_access_token_here'
+  -d '{ "expected_date": "2022-09-22", "line_items": [ { "product_uid": "52781", "expected_quantity": 8, "keep_serial_numbers": true, "has_expiration_date": false, "storage_conditions": "2" } ] }'
+</pre>
+
+<%= render_recording :merchants_fbs_purchase_orders_create %>
+
```

Comment:
How about adding a link to the appendix here? It took me a little bit to realise there is another section below explaining some fields in more detail (like storage_conditions).

Something like: `View purchase order creation options in more detail`

Reply:
[fixed](https://github.com/skroutz/developer.skroutz.gr/pull/192/commits/40680b2c86bcf8f2607cbf2a556ca06abb27b7e9)